test/adoption_center/test_client.rb

```diff
@@ -0,0 +1,18 @@
+require 'minitest/autorun'
+require '../lib/client'
```

Comment:
So that you don't have to require all files in the `lib/` folder you can have Rails autoload them, doing something like this on you `config/application.rb` file `config.autoload_paths << Rails.root.join('lib')`.